allauth/socialaccount/providers/openid_connect/urls.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import itertools
+
+from django.urls import include, path
+
+from allauth.socialaccount.providers.oauth2.views import (
+    OAuth2CallbackView,
+    OAuth2LoginView,
+)
+from allauth.socialaccount.providers.openid_connect.provider import (
+    provider_classes,
+)
+from allauth.socialaccount.providers.openid_connect.views import (
+    OpenIDConnectAdapter,
+)
+
+
+def _factory(_provider_id):
+    class OpenIDConnectAdapterServer(OpenIDConnectAdapter):
+        provider_id = _provider_id
+
+    return OpenIDConnectAdapterServer
+
+
+def default_urlpatterns(provider):
+    adapter_class = _factory(provider.id)
+    urlpatterns = [
+        path(
+            "login/",
+            OAuth2LoginView.adapter_view(adapter_class),
+            name=provider.id + "_login",
+        ),
+        path(
+            "login/callback/",
+            OAuth2CallbackView.adapter_view(adapter_class),
+            name=provider.id + "_callback",
+        ),
+    ]
+
+    return [path(provider.get_slug() + "/", include(urlpatterns))]
```

Comment:
I am not sure we should nest OIDC based providers below a  `/accounts/oidc/` path. The fact that such providers are using this subprovider mechanism is just an implementation detail that better not shine through on the outside.  Not having the "/oidc/" in between would make it easier to move to (or perhaps even away from) an OIDC based subprovider.

Reply:
I have removed the `/oidc` component from the OpenID Connect URL paths.

Reply:
Note that this is a change to the OpenID Connect provider's `get_slug` method via https://github.com/pennersr/django-allauth/pull/3165/commits/cc77a03aecf1ad68caac10753bba40e2ed075318.